hikari/api/rest.py

```diff
@@ -306,6 +307,10 @@ async def edit_channel(
             This should be either 60, 1440, 4320 or 10080 seconds and, as of
             writing, ignores the parent channel's set default_auto_archive_duration
             when passed as `hikari.undefined.UNDEFINED`.
+        applied_tags : hikari.undefined.UndefinedOr[typing.Sequence[hikari.channels.ForumTag]]
+            If provided, the new tags applied to the thread.
```

Comment:
Slight nitpick.
```suggestion
            If provided, the new tags to apply to the thread.
```
Also, it might be worth removing the spacing between the following statement for consistency with the docstring in the edit method of GuildChannel.

Reply:
Are you referring to the spacing in this section?
```
If provided, the new auto archive duration for this thread. This
only applies to threads.

This should be either 60, 1440, 4320 or 10080 seconds, as of
writing.
```

Reply:
Lines 310-313:
```
applied_tags : hikari.undefined.UndefinedOr[typing.Sequence[hikari.channels.ForumTag]]
    If provided, the new tags applied to the thread.

    This only applies to threads in a forum channel.
```
Becomes:
```
applied_tags : hikari.undefined.UndefinedOr[typing.Sequence[hikari.channels.ForumTag]]
    If provided, the new tags applied to the thread. This only applies
    to threads in a forum channel.
```

Reply:
Ah, yep, I see. I went ahead and made all the new docs more consistent between `RESTClient.edit_channel` and `GuildChannel.edit`.